theforeman/foreman-ansible-modules
# BZ#2118790 - Convert2RHEL - Content views for activation keys

Add content views for Convert2RHEL activation keys
with enabled repositories only for the specific OS

**Steps to test**
- install centos7
- update host
- reboot host so that new kernel is loaded
- register host to satellite with "CentOS 7 converting" host group
- Run `yum install convert2rhel`

For more details see https://bugzilla.redhat.com/show_bug.cgi?id=2118790
